src/include/daos_srv/evtree.h

```diff
@@ -760,6 +760,13 @@ int evt_iter_empty(daos_handle_t ih);
  */
 int evt_iter_delete(daos_handle_t ih, struct evt_entry *ent);
 
+/**
+ * Mark the record at the current cursor as being corrupt.
+ *
+ * \param ih		[IN]	Iterator open handle.
```

Comment:
This is not doxygen format, when we find code using this notation (and there is still a lot) then we should be migrating it to actual doxygen format.
```suggestion
 * \param[in]	ih		Iterator open handle.
```